crudbuilder/abstract.py

```diff
@@ -78,7 +79,10 @@ def check_permission_required(self):
     @property
     def get_inlineformset(self):
         if self.crud.inlineformset:
-            return self.crud.inlineformset().construct_formset()
+            if isinstance(self.crud.inlineformset, list):
```

Comment:
@meska about your **inlineformset changes**
i wanna create an inlineformset with 2+ child model(inline_model) with the same Parent...
could your feature permit me this task?

Example:
+  Parent form
     + child form(same parent)
     + child form(same parent)